crypto/bn/bn_print.c

```diff
@@ -138,10 +138,10 @@ int BN_hex2bn(BIGNUM **bn, const char *a)
         a++;
     }
 
-    for (i = 0; i <= (INT_MAX/4) && isxdigit((unsigned char)a[i]); i++)
+    for (i = 0; i <= INT_MAX / 4 && ossl_isxdigit(a[i]); i++)
```

Comment:
I'll do a second reformatting commit for this file -- there are plenty of similar cases present.